src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -54,6 +54,7 @@
                                     SettingOverrideSource="{x:Bind Appearance.DarkColorSchemeNameOverrideSource, Mode=OneWay}">
                 <ComboBox ItemsSource="{x:Bind Appearance.SchemesList, Mode=OneWay}"
                           SelectedItem="{x:Bind Appearance.CurrentColorScheme, Mode=TwoWay}"
+                          Padding="4"
                           Style="{StaticResource ComboBoxSettingStyle}">
                     <ComboBox.ItemTemplate>
                         <DataTemplate x:DataType="local:ColorSchemeViewModel">

```

Comment:
i was thinking more like, we could reduce the inner padding inside the grid to generally tighten it up :)

Reply:
nah, it looks fine. let's ship it.